packages/ui/components/form/select/Select.tsx

```diff
@@ -46,12 +47,21 @@ export const Select = <
             props.classNames?.option
           ),
         placeholder: (state) =>
-          classNames("text-gray-400 text-sm dark:text-darkgray-400", state.isFocused && "hidden"),
+          classNames(
+            "text-gray-400 text-sm dark:text-darkgray-400",
+            state.isFocused && variant !== "checkbox" && "hidden"
```

Comment:
Multicheckbox shows the total number of items selected - placeholder if 0 

So this isnt needed